go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/lib/pq v1.10.3
 	github.com/manicminer/hamilton v0.44.0
 	github.com/masterzen/winrm v0.0.0-20200615185753-c42b5136ff88
-	github.com/mattn/go-isatty v0.0.17
+	github.com/mattn/go-isatty v0.0.20
```

Comment:
I assume that the `go-isatty` and `golang.org/x/sys` version bumps are unrelated to the CVE, is that right?

Reply:
Unrelated to the CVE, but brought by the update:

```
$ go get github.com/hashicorp/go-retryablehttp@v0.7.7
go: upgraded github.com/fatih/color v1.15.0 => v1.16.0
go: upgraded github.com/hashicorp/go-retryablehttp v0.7.4 => v0.7.7
go: upgraded github.com/mattn/go-isatty v0.0.17 => v0.0.20
go: upgraded golang.org/x/sys v0.18.0 => v0.20.0
```